<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1439</width>
    <height>852</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>BuscaCursos - 2020-1</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QGraphicsView" name="graphicsView_7"/>
    </item>
    <item row="1" column="2">
     <widget class="QGraphicsView" name="graphicsView_6"/>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView_4"/>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView_1"/>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="graphicsView_5"/>
    </item>
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicsView_3"/>
    </item>
   </layout>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>790</y>
      <width>112</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
